src/store/MarkerStore.js

```diff
@@ -160,7 +160,8 @@ export default class MarkerStore {
 
         // Y axis positioning logic
         if (this.yPositioner.toLowerCase() === 'none') {
-            this.bottom = undefined;
+            // set the bottom value if there's a bottomWidget, else set it to undefined
+            this.bottom = this.stx.chart.yAxis.initialMarginBottom !== 50 ? 130 : undefined;
```

Comment:
@kellybinary it'd be better to use `this.stx.chart.yAxis.initialMarginBottom` to set `this.bottom` value.

Reply:
@morteza-binary 
<img width="751" alt="Screenshot 2019-05-08 at 16 27 47" src="https://user-images.githubusercontent.com/24283918/57361219-85875d80-71ae-11e9-96e1-46dce5dad116.png">

using ```this.stx.chart.yAxis.initialMarginBottom``` as `this.bottom` value will cause the the end marker line overlapping with the spot too